city_scrapers/spiders/chi_northern_il_university.py

```diff
@@ -0,0 +1,134 @@
+from city_scrapers_core.constants import BOARD, COMMITTEE, NOT_CLASSIFIED
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+import scrapy
+import datetime
+import re
+
+month_mapping = {
+    "January"  : 1,
+    "February" : 2,
+    "March"    : 3,
+    "April"    : 4,
+    "May"      : 5,
+    "June"     : 6,
+    "July"     : 7,
+    "August"   : 8,
+    "September": 9,
+    "October"  : 10,
+    "November" : 11,
+    "December" : 12,
+}
+
+
+class ChiNorthernIlUniversitySpider(CityScrapersSpider):
+    name = "chi_northern_il_university"
+    agency = "Northeastern Illinois University"
+    timezone = "America/Chicago"
+    start_urls = ["https://www.neiu.edu/about/board-of-trustees/calendar-of-meetings"]
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.material_map = dict()
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs. 
+        """
+
+        self._parse_links()
+        yield None
+        '''
+        items_list = response.xpath('//div[@class="field field--name-field-generic-body field--type-text-long field--label-hidden field--item"]')
+        year_list = items_list.xpath('.//h2')
+        for years in year_list.extract():
+            #print(years)
+            year = re.findall('\d{4}',years)[0]
+            items = year_list.xpath('.//following-sibling::ul[1]/li').extract()
+            for item in items:
+                #print(item)
+                meeting = Meeting(
+                    title=self._parse_title(item),
+                    description='',
+                    classification=self._parse_classification(item),
+                    start=self._parse_start(item,year),
+                    end=None,
+                    #all_day=self._parse_all_day(item),
+                    time_notes="See agenda for meeting time",
+                    location=self._parse_location(item),
+                    #links=self._parse_links(item),
+                    #source=self._parse_source(response),
+                )
+
+                meeting["status"] = self._get_status(meeting)
+                meeting["id"] = self._get_id(meeting)
+                yield meeting'''
+
+    def _parse_title(self, item):
+
+        """Parse or generate meeting title."""
+        pattern= re.compile('-(.*(Committee|Meeting))',re.IGNORECASE)
+        title_list = re.findall(pattern, item)
+        if title_list == None:
+            return 'Board of Trustees'
+        title, ign=title_list[-1]
+        return title.replace("<strong>","").replace("</strong>","").replace("\xa0"," ")
```

Comment:
We should avoid handing HTML tags as strings if we can. We have a good amount of examples of parsing out strings with CSS selectors using `::text` and XPath with `/text()`

Reply:
Thanks a lot. I'll improve it once I'm available.